swan/Dockerfile

```diff
@@ -0,0 +1,162 @@
+# Installs the extensions and packages required by SWAN
+# to adapt the base alma9 notebook image to SWAN.
+
+# FROM gitlab-registry.cern.ch/swan/docker-images/notebook
+FROM gitlab-registry.cern.ch/pesteves/imageregistry:notebook
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+
+RUN echo "Building systemuser image with tag ${VERSION_DOCKER_IMAGE}"
+
+# Switch to superuser to install packages
+USER root
+
+# Install required dnf packages
+RUN dnf install -y  git \
+   # gcc \
+   # gcc-c++ \
+   # kernel-devel \
+   # libcurl-openssl-devel \  -- Does not exist in dnf repos
+   # libffi-devel \
+   # ncurses-devel \
+    nano \
+    nodejs \
+    patch \
+   # sqlite-devel \
+    unzip \
+    which \
+   # zeromq3-devel \ -- Does not exist in dnf repos
+    perl-Digest-MD5 \
+    texlive \
+    fontconfig
+
+# Install HEP_OSlibs
+# This installs system packages for LCGs to work
+RUN dnf -y install https://linuxsoft.cern.ch/wlcg/el9/x86_64/wlcg-repo-1.0.0-1.el9.noarch.rpm
+RUN dnf -y install HEP_OSlibs-9.1.0-2.el9
+
+# TODO: Move spark related packages to the third image
+
+# Install required packages
+RUN dnf install -y \
+    # Required by ROOT
+    libXft \
+    # Required by matplotlib
+    tk \
+    # Useful to monitor resource consumption
+    htop \
+    # Install Cloudera dependencies - required by IT Spark clusters
```

Comment:
3rd image? No longer needed? Or lcg?

Reply:
It is also not needed anymore, so it was removed.